nbs/models.timellm.ipynb

```diff
@@ -68,7 +68,7 @@
     "from neuralforecast.losses.pytorch import MAE\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #157.                try:
</pre><p>This pattern is very awkward. Will it not look for local files first and only download if necessary?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Thank you for your comment. However, I believe there might be a misunderstanding here. This code is not actually downloading or loading any models. It's simply trying to import classes from the transformers library and checking if the library is installed.
The pattern used here is a common way to check for the availability of an optional dependency. It doesn't involve looking for local files or downloading anything. The actual model loading and potential downloading would happen later when these classes are used to instantiate models.
If you have concerns about how models are loaded or downloaded, that would be more relevant to the code where we actually use these classes to create model instances. Would you like to discuss that part of the code instead?

Reply:
The comment was made on the reviewnb app, not where github is showing it. I meant all the blocks where the only thing that changes is the `local_files_only` argument

Reply:
I've made the changes based on your feedback.